src/components/ConnectedSample/ConnectedSample.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { ReduxState } from 'src/reducers/rootReducer';
+import uuid from 'uuid/v4';
+
+import { fillSampleAction } from 'src/actionCreators/sample/sample';
+import List from './List';
+import { Dispatch, bindActionCreators } from 'redux';
+
+export const ConnectedSample: React.FC<{ initial: string; fillSampleAction: typeof fillSampleAction }> = props => {
+  const handleAddNewSample = (event: React.MouseEvent<HTMLButtonElement>) => {
+    props.fillSampleAction({
```

Comment:
Couldn't get test to pass when using useDispatch without using store and mount instead of shallow rendering.